### DISCUSSION TITLE
filtering using headerRender

### DISCUSSION THREAD

**Discussion by hatem-marzougui** [Comcast/react-data-grid#2908] [General]
i use React JS , and i didnt understand how to add headerFiltering . 
from where can i import < FilterRenderer >
i would like to do something like that 
![Screenshot from 2022-05-11 16-45-21](https://user-images.githubusercontent.com/66776531/167892170-3786ddd1-5245-4213-92d8-c8ab6646ce31.png)
@nstepien 
@amanmahajan7

**Comment by rohit-squirrel** [comment]
Same issue for me as well.
Using data-grig version 7.0.0-canary.30

**Comment by godon019** [comment]
here https://github.com/adazzle/react-data-grid/blob/main/website/demos/HeaderFilters.tsx is the example code. hope it helps
